src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -564,17 +577,17 @@ private void updateComponents() {
                 }
             } else {
                 setDescription(Localization
-                                           .lang("Please enter a search term. For example, to search all fields for <b>Smith</b>, enter:<p>"
-                                                 + "<tt>smith</tt><p>"
-                                                 + "To search the field <b>Author</b> for <b>Smith</b> and the field <b>Title</b> for <b>electrical</b>, enter:<p>"
-                                                 + "<tt>author=smith and title=electrical</tt>"));
+                        .lang("Please enter a search term. For example, to search all fields for <b>Smith</b>, enter:<p>"
```

Comment:
Why don't you directly change this to a TextFlow instead of the replacement?  As far as I see the text is fixed, so no need to dynamically replace it.

e.g. create multiple TextFlow objects for each part of the string and then add them to a FlowPane:
This was my approach in one of the OpenOffice dialogs: 
```
        Text startText = new Text(start);
        Text inBetweenText = new Text(inBetween);
        inBetweenText.setStyle("-fx-font-weight: bold");
        Text endText = new Text(end);
       FlowPane flow = new FlowPane(startText, inBetweenText, endText);
```

Reply:
This should be also possible in the other case. Anoter idea would be to reuse the methods we have in TooltipTextUtil or extend it with a normal method:

Reply:
I thought about doing that but then you would need to break up every localized string into smaller substrings. And i didn't want to touch the localization.
Or am i misunderstanding what you are saying?

Is there any reason why you would want it to be a FlowPane instead of a TextFlow element? 


And your second suggestion is to put the createFormattedDescription method into the TooltipTextUtil class?

Thanks for your feedback. This is my first pr for a public repository so thank you for helping.

Reply:
Okay, if you don't want to touch it it's fine. 
Regarding the Tooltip Class. For the search tooltips someone created that class and we also have tests in place for it so the idea is to simply rename that class to TextUtils or simliar and add a method which works for normal texts.

TextFlow should be fine, in the openOffice case I had to use a FlowPane cause the text was inside a table.